produce/lib/produce/cloud_container.rb

```diff
@@ -0,0 +1,89 @@
+require 'spaceship'
+require_relative 'module'
+
+module Produce
+  class CloudContainer
+    def create(options, _args)
+      login
+
+      ENV["CREATED_NEW_CLOUD_CONTAINER_ID"] = Time.now.to_i.to_s
+
+      container_identifier = options.container_identifier || UI.input("iCloud Container identifier: ")
+
+      if container_exists?(container_identifier)
+        UI.success("[DevCenter] iCloud Container '#{options.container_name} (#{options.container_identifier})' already exists, nothing to do on the Dev Center")
+        ENV["CREATED_NEW_CLOUD_CONTAINER_ID"] = nil
+        # Nothing to do here
+      else
+        container_name = options.container_name || container_identifier.gsub('.', ' ')
+
+        UI.success("Creating new iCloud Container '#{container_name}' with identifier '#{container_identifier}' on the Apple Dev Center")
+
+        container = Spaceship.cloud_container.create!(identifier: container_identifier, name: container_name)
+
+        if container.name != container_name
+          UI.important("Your container name includes non-ASCII characters, which are not supported by the Apple Developer Portal.")
+          UI.important("To fix this a unique (internal) name '#{container.name}' has been created for you.")
+        end
+
+        UI.message("Created iCloud Container #{container.cloud_container}")
+
+        UI.user_error!("Something went wrong when creating the new iCloud Container - it's not listed in the iCloud Container list") unless container_exists?(container_identifier)
+
+        ENV["CREATED_NEW_CLOUD_CONTAINER_ID"] = Time.now.to_i.to_s
```

Comment:
Yeah, this should be removed. I had some sort of idea that the user could use this variable to check if the container was created, but never took it all the way.. :)
Since I never had to check this variable after I've used this method, I'll remove it.